api/v1/_submissions/PKPBackendSubmissionsController.php

```diff
@@ -430,6 +439,70 @@ public function delete(Request $illuminateRequest): JsonResponse
         return response()->json([], Response::HTTP_OK);
     }
 
+    /**
+     * Delete a list of incomplete submissions
+     */
+    public function bulkDeleteIncompleteSubmissions(Request $illuminateRequest): JsonResponse
+    {
+        $submissionIdsRaw = paramToArray($illuminateRequest->query('ids') ?? []);
+
+        if (empty($submissionIdsRaw)) {
+            return response()->json([
+                'error' => __('api.submission.400.missingQueryParam'),
+            ], Response::HTTP_BAD_REQUEST);
+        }
+
+        $submissionIds = [];
+
+        foreach ($submissionIdsRaw as $id) {
+            $integerId = intval($id);
+
+            if (!$integerId) {
+                return response()->json([
+                    'error' => __('api.submission.400.invalidId', ['id' => $id])
+                ], Response::HTTP_BAD_REQUEST);
+            }
+
+            $submissionIds[] = $id;
+        }
+
+        $submissions = $this->getSubmissionCollector($illuminateRequest->query())
+            ->filterBySubmissionIds($submissionIds)
+            ->filterByIncomplete(true)
+            ->getMany()
+            ->all();
+
+        $submissionIdsFound = array_map(fn (Submission $submission) => $submission->getData('id'), $submissions);
+
+        if (array_diff($submissionIds, $submissionIdsFound)) {
+            return response()->json([
+                'error' => __('api.404.resourceNotFound')
+            ], Response::HTTP_NOT_FOUND);
+        }
+
+        $context = $this->getRequest()->getContext();
+
+        foreach ($submissions as $submission) {
+            if ($context->getId() != $submission->getData('contextId')) {
+                return response()->json([
+                    'error' => __('api.submissions.403.deleteSubmissionOutOfContext'),
+                ], Response::HTTP_FORBIDDEN);
+            }
+
+            if (!Repo::submission()->canCurrentUserDelete($submission)) {
+                return response()->json([
+                    'error' => __('api.submissions.403.unauthorizedDeleteSubmission'),
+                ], Response::HTTP_FORBIDDEN);
+            }
+        }
+
+        foreach ($submissions as $submission) {
+            Repo::submission()->delete($submission);
+        }
+
+        return response()->json([], Response::HTTP_OK);
```

Comment:
Perhaps the frontend won't be very friendly with that, but the `response()->noContent()` would fit better :)